auth-web/src/components/auth/CreateAccount/AccountCreatePremium.vue

```diff
@@ -230,7 +230,7 @@ export default class AccountCreatePremium extends Mixins(Steppable) {
     var org: Organization = {
       id: this.currentOrganization.id,
       name: details.bcolAccountDetails.orgName,
-      accessType: Account.PREMIUM,
+      accessType: this.currentOrganization.accessType,
```

Comment:
The initial code was wrong. It will set the access type as PREMIUM always if the account is PREMIUM. Same component is used in update as well. So in create, backend will populate the access type.